R/databricks_connection.R

```diff
@@ -0,0 +1,64 @@
+databricks_connection <- function(config, extensions) {
+  tryCatch({
+    callSparkR <- get("callJStatic", envir = asNamespace("SparkR"))
```

Comment:
Granted this is not technically an "option" but it's the one shared sink that the R environment provides to store references to functions in a decoupled way (we have used this frequently to allow R packages to call back into RStudio). Javier, if you wanted to make it generic you could establish a standard that the following would be called:

```r
options(sparklyr.remote_gateway = function() {})
```

Then Databricks could set that value in their environment, but other environment that provide other remote gateways could set it to something different (this might dovetail well with your idea of supporting a generic `method = "remote"` option).